### PR TITLE
Update sharepoint documents configurepermissions-document

### PR DESCRIPTION
**Pull request by fbhub2**
Update sharepoint documents configurepermissions-document with details about Confidential activities and Visible for

### CODE REVIEW

File: docs/en/document/cloud/sharepoint-documents/configure-permissions-documents.md [hunk 1]
```diff
@@ -52,6 +64,8 @@ Make sure there are no user accounts in this group. Only service accounts.​
 ​![Review SharePoint site groups​ -screenshot][img4]
 
 <!-- Referenced links -->
+[1]: https://help.superoffice.com/Documentation/Help/EN/CRM/10.1/WebHelpAdmin/chap07/Adding_users_web.htm
```
Comment (digitaldiina) [inline docs/en/document/cloud/sharepoint-documents/configure-permissions-documents.md:67]: We've migrated the admin guide to docs. Suggest you link directly to the most recent version of the **Add user** page.

```suggestion
[1]: ../../../admin/user-management/learn/add-associate.md
```

File: docs/en/document/cloud/sharepoint-documents/configure-permissions-documents.md [hunk 2]
```diff
@@ -13,6 +13,18 @@ so.client: online
...
 
+> [!NOTE]
+> This feature is only relevant if 'Confidential activities' is enabled for users in 'Settings and maintenance', and 'Visible for' field is set on the the document property:
+
+<details>
+<summary>Show details for Confidential activities and Visible for</summary>
+
+'Confidential activities' in the [Settings and maintenance - Users][1]: activates the 'Visible for' for the user.
+
+'Visible for' field in the [document dialog][2]: display a list that allows you to choose whether the document will be visible to all users, private (i.e. visible only to the user defined as owner), or visible to all users in one of the user groups that the owner belongs to.
```
Comment (digitaldiina) [inline docs/en/document/cloud/sharepoint-documents/configure-permissions-documents.md:24]:
```suggestion
* The **Visible for** field in the [document dialog][2] display a list that allows you to choose whether the document will be visible to all users, private (visible only to the user defined as owner), or visible to all users in one of the user groups that the owner belongs to. 
```

File: docs/en/document/cloud/sharepoint-documents/configure-permissions-documents.md [hunk 3]
```diff
@@ -13,6 +13,18 @@ so.client: online
 
 # How to set up SharePoint permissions for documents
 
+> [!NOTE]
+> This feature is only relevant if 'Confidential activities' is enabled for users in 'Settings and maintenance', and 'Visible for' field is set on the the document property:
+
+<details>
+<summary>Show details for Confidential activities and Visible for</summary>
+
+'Confidential activities' in the [Settings and maintenance - Users][1]: activates the 'Visible for' for the user.
```
Comment (digitaldiina) [inline docs/en/document/cloud/sharepoint-documents/configure-permissions-documents.md:22]:
```suggestion
* **Confidential activities** in the [Settings and maintenance - Users][1] activates the *Visible for* for the user.
```

File: docs/en/document/cloud/sharepoint-documents/configure-permissions-documents.md [hunk 4]
```diff
@@ -13,6 +13,18 @@ so.client: online
 
 # How to set up SharePoint permissions for documents
 
+> [!NOTE]
+> This feature is only relevant if 'Confidential activities' is enabled for users in 'Settings and maintenance', and 'Visible for' field is set on the the document property:
```
Comment (digitaldiina) [inline docs/en/document/cloud/sharepoint-documents/configure-permissions-documents.md:17]:
```suggestion
> This feature is relevant only if **Confidential activities** is enabled for users in Settings and maintenance, and the **Visible for** field is set on the document property:
```

File: docs/en/document/cloud/sharepoint-documents/configure-permissions-documents.md [hunk 5]
```diff
@@ -13,6 +13,18 @@ so.client: online
...
+
+<details>
+<summary>Show details for Confidential activities and Visible for</summary>
```
Comment (digitaldiina) [inline docs/en/document/cloud/sharepoint-documents/configure-permissions-documents.md:20]: We aim to use this syntax less and I recommend omitting it. If it's worth mentioning, there's no need to hide the info.

**Review by digitaldiina — CHANGES_REQUESTED** [review]
@pavlas77 A few suggestions from me. Click **Accept** if you agree. Minimum change is to not use single quotes for emphasis.

File: docs/en/document/cloud/sharepoint-documents/configure-permissions-documents.md (hunk 5 above)
Comment (pavlas77) [reply docs/en/document/cloud/sharepoint-documents/configure-permissions-documents.md]: @digitaldiina  It was Frode's edits. I just asked you for a review. 

Anyway, I changed the <details> section to be visible and part of the Note. Is it ok now? 

Thanks for the info.

**Review by digitaldiina — APPROVED** [review]

**Review by digitaldiina — APPROVED** [review]

File: docs/en/document/cloud/sharepoint-documents/configure-permissions-documents.md (hunk 5 above)
Comment (digitaldiina) [reply docs/en/document/cloud/sharepoint-documents/configure-permissions-documents.md]: Thanks @pavlas77 and @fbhub2 :)

**Pull request merged**